common/nodejs-utils/src/telemetry.rs

```diff
@@ -3,29 +3,44 @@ use std::{
     path::Path,
 };
 
-use opentelemetry::{
-    global::{self, BoxedTracer},
-    KeyValue,
-};
+use opentelemetry::{global, KeyValue};
 use opentelemetry_sdk::{
     trace::{Config, TracerProvider},
     Resource,
 };
 
-pub fn init_tracer(buildpack_name: impl Into<String>) -> BoxedTracer {
+pub fn init_tracing(buildpack_name: impl Into<String>) -> TracerProvider {
     let bp_name = buildpack_name.into();
     let telem_file_path = Path::new("/tmp")
         .join("cnb-telemetry")
         .join(format!("{}.jsonl", bp_name.replace('/', "_")));
+
+    // Ensure the telemetry dir exists
     if let Some(parent_dir) = telem_file_path.parent() {
         let _ = create_dir_all(parent_dir);
     }
-    let exporter = match File::create(telem_file_path) {
+
+    // Create the telemetry file if it doesn't exist
+    let _ = File::options()
```

Comment:
@joshwlewis to append for github output in our release actions i use:

```
OpenOptions::new()
  .append(true)
  .open(filename)
```

Reply:
Yeah, I'm doing that here: https://github.com/heroku/buildpacks-nodejs/pull/659/files#diff-ebe034d4507bc0c116307c3ec6c58a708d461c5f218af356d27bd438542d5b9fR31. I was trying to separate steps due to issues I was seeing.